src/routes/events/event.py

```diff
@@ -32,3 +35,41 @@ def latest_event():
     data = connection()
     latest = data[-1]
     return latest
+
+
+@event_bp.route("/announcement", methods=["POST"])
+def make_announcement():
+    API_KEY = os.getenv("API_KEY")
+    WEBHOOK_URL = os.getenv("WEBHOOK_URL")
+
+    api_key = request.args.get("api_key")
+
+    if not (api_key == API_KEY) or not API_KEY:
+        return "Unauthorized: Invalid API Key", 401
+
+    if not WEBHOOK_URL:
+        return (
+            "Internal Error: No Webhook URL is configured in environment variables",
+            500,
+        )
+
+    data = connection()
+    post_data = request.json
+
+    if not post_data or not post_data.get("event_id"):
+        return "Bad Request: No Event ID is found", 400
+
+    event_id = post_data["event_id"]
+    required_event = None
+
+    for event in data:
+        if event["id"] == event_id:
+            required_event = event
+            break
+
+    if not required_event:
+        return f"No event found with ID: {event_id}", 404
+
+    send_discord_announcement(WEBHOOK_URL, required_event)
+
+    return jsonify(required_event)
```

Comment:
Return success/failed message instead of data.

Reply:
Handled in cd6301f